wheels/build/wheels.yml

```diff
@@ -62,6 +62,20 @@ packages:
         prebuild:
             wheel: for f in sqlite3.h sqlite3.c sqlite3ext.h; do cp ${SRC_ROOT_1}/$f $f || return 1; done && echo 'include_dirs=.' >>${SRC_ROOT_0}/setup.cfg
         build_args: build_static bdist_wheel
+    python-ldap:
+        version: 2.4.27
+        src:
+            - http://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz
+            - https://pypi.python.org/packages/fc/99/9eed836fe4d916792994838df125da9c25c5f7c31abfbf6f0ab076e5f419/python-ldap-2.4.27.tar.gz
```

Comment:
I *think* (from a quick look at the code) the build is launched from the directory of the last src, so without this it tries to build python-ldap from the openldap dir and fails.